N24DCCN058/bai tap buoi 3/Bai4.cpp

```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+using namespace std ;
+int main(){
+    int a = 10 , b = 11 ;
+    b -= a ; 
+    a += b ; 
+    b = a - b ; 
+    cout << a << " " << b ; 
```

Comment:
hay nhể, có 1 bài khác biệt các bài khác đỡ buồn ngủ hẳn:)

Reply:
Đúng phải k anh, sợ nha =)))

Reply:
> Đúng phải k anh, sợ nha =)))

uk đúng ròi
tại a thấy tự nhiên có người làm cách độc lạ ko như mấy bạn khác nên bất ngờ thôi
cấp 3 em chuyên toán à, toàn kiếm cái khó làm mới chịu:)